src/index.ts

```diff
@@ -125,4 +127,8 @@ export {
   IConfigurationWire,
   IPrecomputedConfigurationResponse,
   PrecomputedFlag,
+
+  // Test helpers
+  setSaltOverrideForTests,
+  decodePrecomputedFlag,
```

Comment:
Used in a test [here](https://github.com/Eppo-exp/node-server-sdk/pull/89/files#diff-18466f9d870e2151722b884f1cb0df6c1530517bde9468ec2a5abbbb8f655210R565) and [here](https://github.com/Eppo-exp/node-server-sdk/pull/89/files#diff-18466f9d870e2151722b884f1cb0df6c1530517bde9468ec2a5abbbb8f655210R576)

To verify that an initialized node client exports correctly obfuscated precomputed flags